### PR TITLE
GDB Command for elapsed time since start of recording

### PR DESCRIPTION
**Pull request by WesleyYue**
Implements #2126 

I'm running the tests locally right now but it's taking a while so I'm going to open this PR for now and report back on the results of the tests.

### CODE REVIEW

File: src/test/elapsed_time.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+send_gdb('b breakpoint')
+expect_gdb('Breakpoint 1')
+
+send_gdb('elapsed-time')
+
+# first group captures numbers >= 1.0
+# second group captures numbers >= 10.0
+expect_gdb('^(?:[1-9]\.\d*|\d\d+\.\d*)$')
```
Comment (rocallahan) [inline src/test/elapsed_time.py]: Would be a bit cleaner to capture the floating point value, convert to float, and then compare it to 1.

File: src/GdbCommand.cc [hunk 2]
```diff
@@ -19,9 +19,8 @@ static SimpleGdbCommand elapsed_time(
       }
 
       ReplayTask* replay_t = static_cast<ReplayTask*>(t);
-      double elapsed_time = (
-          replay_t->current_trace_frame().monotonic_time() -
-          replay_t->session().get_trace_start_time());
+      double elapsed_time = (replay_t->current_trace_frame().monotonic_time() -
+                             replay_t->session().get_trace_start_time());
```
Comment (rocallahan) [inline src/GdbCommand.cc]: You can remove the outer parentheses.

**Comment by rocallahan** [comment]
I'd prefer if you squash these commits into one for this review.

**Comment by rocallahan** [comment]
and thanks!!

**Comment by WesleyYue** [comment]
Have you had the chance to review the new changes yet?

**Comment by rocallahan** [comment]
Sorry... Github never sends me emails about updates to pull requests :-(.

Thanks for the patch!

**Comment by WesleyYue** [comment]
Thanks for all your help with the PR! :)

**Pull request merged**